cmd/vulcan-repository-sctrl/custom-rules/repository-with-security-control.yaml

```diff
@@ -0,0 +1,100 @@
+rules:
+  - id: repository-with-lava-scan
+    message: 'Action using lava scan binary'
+    severity: INFO
+    languages:
+      - yaml
+    paths:
+      include:
+        - ".github/**/*.yaml"
+        - ".github/**/*.yml"
+    patterns:
+      - pattern-regex: 'lava scan'
+  - id: repository-with-lava-action
+    message: 'Action running Lava'
+    severity: INFO
+    languages:
+      - yaml
+    paths:
+      include:
+        - ".github/**/*.yaml"
+        - ".github/**/*.yml"
+    patterns:
+      - patterns:
+          - pattern: |
+              ...
+              uses: "$USES"
+          - metavariable-pattern:
+              metavariable: $USES
+              language: generic
+              pattern-either:
+                - pattern: lava-internal-action
+                - pattern: lava-action
+  - id: repository-with-sonarqube-action
+    message: 'Action running Sonarqube'
+    severity: INFO
+    languages:
+      - yaml
+    paths:
+      include:
+        - ".github/**/*.yaml"
+        - ".github/**/*.yml"
+    patterns:
+      - pattern: |
+          ...
+          uses: "$USES"
+      - metavariable-pattern:
+          metavariable: $USES
+          language: generic
+          pattern: code-quality-action
+  - id: repository-with-trivy-action
+    message: 'Action running Trivy'
+    severity: INFO
+    languages:
+      - yaml
+    paths:
+      include:
+        - ".github/**/*.yaml"
+        - ".github/**/*.yml"
+    patterns:
+      - pattern: |
+          ...
+          uses: "$USES"
+      - metavariable-pattern:
+          metavariable: $USES
+          language: generic
+          pattern: trivy-action
+  - id: repository-with-snyk-action
+    message: 'Action running Snyk'
+    severity: INFO
+    languages:
+      - yaml
+    paths:
+      include:
+        - ".github/**/*.yaml"
+        - ".github/**/*.yml"
+    patterns:
+      - pattern: |
+          ...
+          uses: "$USES"
+      - metavariable-pattern:
+          metavariable: $USES
+          language: generic
+          pattern: snyk/actions
+  - id: repository-with-govulncheck
+    message: 'Action running Govulncheck'
+    severity: INFO
+    languages:
+      - yaml
+    paths:
+      include:
+        - ".github/**/*.yaml"
+        - ".github/**/*.yml"
+    patterns:
+      - pattern: |
+          ...
+          uses: "$USES"
+      - metavariable-pattern:
+          metavariable: $USES
+          language: generic
+          pattern: govulncheck-action
```

Comment:
I think it is quite common to install and run the govulncheck command. So, I might also check if the binary is run.

Reply:
I've added another rule to verify if the binary is being called.
Added [here](https://github.com/adevinta/vulcan-checks/pull/806/files#diff-92895a02ca895131904cdd6800f5bf564cef58757c5826fdf73c9b0e18837c62R102-R112).

I've used the most common form of running it in order to make the pattern matching regexp simpler:
```sh
govulncheck ./...
```

I've done a quick analysis and was the only occurrences I found within GH actions.